tutorials/data_provider.md

```diff
@@ -217,17 +217,29 @@ where
         if req.id.locale.region == Some(region!("CH")) {
             if let Ok(mut decimal_payload) = res.payload.dynamic_cast_mut::<DecimalSymbolsV1Marker>() {
                 decimal_payload.with_mut(|data| {
-                    // Change the grouping separator for all Swiss locales to '🐮'
-                    data.grouping_separator = Cow::Borrowed("🐮");
+                    // Change the digit 0 for all Swiss locales to '🐮'
+                    data.digits[0] = '🐮';
                 });
             }
         }
         Ok(res)
     }
 }
 
+// Load some en-US data as a base. Baked is the provider for icu::decimal's builtin
+// "compiled data" that it ships by default.
+let data = icu::decimal::provider::Baked
+    .load(DataRequest {
+        id: DataIdentifierBorrowed::for_locale(&locale!("en-US").into()),
+        ..Default::default()
+    })
+    .unwrap()
+    .payload;
+
+// Make a wrapped provider that modifies Swiss data requests
 let provider = CustomDecimalSymbolsProvider(
-    FixedProvider::<DecimalSymbolsV1Marker>::new_default()
+    // Make a simple data provider that provides the loaded en-US data unconditionally
+    FixedProvider::<DecimalSymbolsV1Marker>::from_payload(data)
```

Comment:
Suggestion: maybe just plug in `icu::decimal::provider::Baked` here instead of `FixedProvider`